src/rvpt/main.cpp

```diff
@@ -4,9 +4,72 @@
 
 #include <imgui.h>
 #include <fmt/core.h>
-
 #include "rvpt.h"
 
+#define TINYOBJLOADER_IMPLEMENTATION // define this in only *one* .cc
+#include "tinyobjloader/tiny_obj_loader.h"
+
+void load_model(RVPT& rvpt, std::string inputfile, int material_id)
+{
+    rvpt.get_asset_path(inputfile);
+
+    tinyobj::attrib_t attrib;
+    std::vector<tinyobj::shape_t> shapes;
+    std::vector<tinyobj::material_t> materials;
+
+    std::string warn;
+    std::string err;
+
+    bool ret = tinyobj::LoadObj(&attrib, &shapes, &materials, &warn, &err, inputfile.c_str());
+
+    if (!warn.empty())
+    {
+        fmt::print("[{}: {}] {}\n", "WARNING", "MODEL-LOADING", warn);
+    }
+
+    if (!err.empty())
+    {
+        fmt::print("[{}: {}] {}\n", "ERROR", "MODEL-LOADING", err);
+    }
+
+    if (!ret)
```

Comment:
This will print whenever an error is encoutered.
This should be conditional on whether err is empty. That way if there is a known error it doesn't print "unknown error" in addition.

Reply:
This handling has been redone in the latest commit